README.md

```diff
@@ -1,2 +1,62 @@
-# ably-labs-template-repo
-This is a template repository to be used for all Ably Labs demos, tools &amp; proof of concepts.
+# Ably Labs Template Repo
+
+---
+
+// This is a template repository to be used for all Ably Labs demos, tools &amp; proof of concepts. Follow these steps to so this repo is easy to use for visitors & maintainers.
+
+1. Update the description of this repo.
+2. Add [topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics) to this repo to clarify the language, tech stack and use case.
+3. Update the [.gitignore](.gitignore) file with one of the [standard templates from GitHub](https://github.com/github/gitignore).
+4. Update [dependabot.yml](.github/dependabot.yml) with the  [configuration for your project](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates).
+5. Replace `REPO_NAME` with the actual name of the repo in the [CONTRIBUTING.md](CONTRIBUTING.md) file.
+6. Update this README so it provides enough information for people to understand how it works, how to run it locally and how it can be deployed to the cloud (see [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes)).
+7. Add a GitHub workflow to build/test/deploy your application. Use the [Ably Control API GitHub action](https://github.com/ably-labs/ably-control-api-action) to avoid creating Ably apps/API keys manually.
+
+Once you're done, remove this section from the README. Good luck! 💪
+
+---
+
+![Place eye candy header image here]()
```

Comment:
it would be nice to use a generic hero image that the person can replace.
just so that this template looks great. perhaps some Alby branding etc.